config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
-  get '/pizzerias/new', to: 'pizzerias#new', as: 'new_pizzeria'
-  post '/pizzerias', to: 'pizzerias#create', as: 'pizzerias'
-  get '/pizzerias/:id', to: 'pizzerias#show', as: 'pizzeria'
-  get '/pizzerias', to: 'pizzerias#index'
+  resources :pizzas
+  # get '/pizzerias/new', to: 'pizzerias#new', as: 'new_pizzeria'
+  # post '/pizzerias', to: 'pizzerias#create', as: 'pizzerias'
+  # get '/pizzerias/:id', to: 'pizzerias#show', as: 'pizzeria'
+  # get '/pizzerias', to: 'pizzerias#index'
```

Comment:
Ewwww

Reply:
Sorry! I realize I completely neglected this file.